renku/core/management/workflow/plan_factory.py

```diff
@@ -693,7 +718,7 @@ def get_indirect_outputs_path(client_path):
 def get_indirect_parameters_path(client_path):
     """Return path to file that contains indirect parameters list."""
     parent = _get_indirect_parent_path(client_path)
-    return parent / "parameters.yml"
+    return parent / "parameters..yml"
```

Comment:
i'm wondering about this change. i mean i see that in the unit test it's being used the very same way, just not sure why the double dots...

Reply:
Ah it was originally `parameters.yml` but for inputs and outputs, it's .txt, so I'd changed to to .txt, then saw that actually it is a yaml file, so changed with back with a search&replace, which I seem to have botched 🙈 
Thanks for catching that!